### ISSUE TITLE
formatting JsonObject inside JsonArray always inlines

### ISSUE THREAD

**Issue by NikkyAI** [falkreon/Jankson#3]
where everything gets indented and formatted nicely, lists/arrays seem to be slightly buggy

example:
```hjson
        "locations": [ 
		{ /* location label for identification */ "label": "local", /* matterbridge gateway identifier */ "gateway": "matterlink", "area": { /* Area type identifier */ "type": "RADIUS", "x": 178, "z": 208, "radius": 50, /* list of dimension ids */ "dimensions": [ -1, 0, 1 ], /* ignores dimension list */ "allDimensions": true }, /* Location outgoing settings */ "outgoing": { /* transmit join events */ "plain": true, /* transmit join events */ "action": true, "advancements": true, /* transmit death messages */ "death": true, /* transmit status updates */ "status": true, /* list of other locations to ignore after handling this */ "skip": [ "default" ] }, /* incoming settings */ "incoming": { /* transmit join events */ "plain": true, /* transmit join events */ "action": true, /* transmit join_leave events */ "join_leave": true, /* receive commands */ "commands": true, /* list of other locations to ignore after handling this */ "skip": [ ] } },
		{ /* location label for identification */ "label": "default", /* matterbridge gateway identifier */ "gateway": "matterlink", "area": { /* Area type identifier */ "type": "INFINITE", /* list of dimension ids */ "dimensions": [ 0, 1 ], /* ignores dimension list */ "allDimensions": true }, /* Location outgoing settings */ "outgoing": { /* transmit join events */ "plain": true, /* transmit join events */ "action": true, "advancements": true, /* transmit death messages */ "death": true, /* transmit status updates */ "status": true, /* list of other locations to ignore after handling this */ "skip": [ ] }, /* incoming settings */ "incoming": { /* transmit join events */ "plain": true, /* transmit join events */ "action": true, /* transmit join_leave events */ "join_leave": true, /* receive commands */ "commands": true, /* list of other locations to ignore after handling this */ "skip": [ ] } },
		{ /* location label for identification */ "label": "management", /* matterbridge gateway identifier */ "gateway": "matterlink", "area": { /* Area type identifier */ "type": "INFINITE", /* list of dimension ids */ "dimensions": [ 0, 1 ], /* ignores dimension list */ "allDimensions": true }, /* Location outgoing settings */ "outgoing": { /* transmit join events */ "plain": false, /* transmit join events */ "action": false, "joins": true, "leaves": true, /* transmit broadcasts */ "say": true, /* list of other locations to ignore after handling this */ "skip": [ ] }, /* incoming settings */ "incoming": { /* transmit join events */ "plain": false, /* transmit join events */ "action": false, /* transmit join_leave events */ "join_leave": true, /* receive commands */ "commands": true, /* list of other locations to ignore after handling this */ "skip": [ ] } }
	],
```

is there a option to enable newlines on objects in Arrays ?

and i need to have these in a list because mapping htem against eg. the label is unordered

**Comment by falkreon** [comment]
fixed by 091281a

**Comment by falkreon** [comment]
There may be room to revisit this, as it de-inlines *all* array members, but without a prior scan through the array's contents this is a good compromise behavior
